packages/core/core/src/services/document-service/utils/populate.ts

```diff
@@ -36,7 +36,7 @@ export const getDeepPopulate = (uid: UID.Schema, opts: Options = {}) => {
       }
 
       case 'media': {
-        acc[attributeName] = { select: ['id'] };
+        acc[attributeName] = { select: ['*'] };
```

Comment:
could you add a note thats important to populate all data so the webhooks contain that informatoin? maybe linking the issue